Create topics (#8)
redhat-developer/terraform-provider-rhoas: docs/resources/topic.md

---
# generated by https://github.com/hashicorp/terraform-plugin-docs
page_title: "rhoas_topic Resource - terraform-provider-rhoas"
subcategory: ""
description: |-

>   rhoas_topic manages a topic from a  Kafka instance in Red Hat OpenShift Streams for Apache Kafka.

Reviewer: ```suggestion
  rhoas_topic manages a topic in a Kafka instance in Red Hat OpenShift Streams for Apache Kafka.
```

---

# rhoas_topic (Resource)

> `rhoas_topic` manages a topic in a  Kafka instance in Red Hat OpenShift Streams for Apache Kafka.

Reviewer 2: ```suggestion
`rhoas_topic` manages a topic in a Kafka instance in Red Hat OpenShift Streams for Apache Kafka.
```

<!-- schema generated by tfplugindocs -->
## Schema

### Required

> - `kafka_id` (String) The number of partition in the topic

Reviewer: ```suggestion
- `kafka_id` (String) The unique ID of the Kafka instance
```

- `name` (String) The name of the Kafka instance

> - `partitions` (Number) The number of partition in the topic

Reviewer 2: ```suggestion
- `partitions` (Number) The number of partitions in the topic
```
